tests/rptest/tests/audit_log_test.py

```diff
@@ -698,24 +744,27 @@ def alter_config(client, values: dict[str, Any], incremental: bool):
             RangeTestItem(
                 f'Attempting delete records for {topic_name}',
                 lambda: self.execute_command_ignore_error(
-                    partial(self.rpk.trim_prefix, topic_name, 0)),
+                    partial(self.super_rpk.trim_prefix, topic_name, 0)),
                 partial(self.api_resource_match, "delete_records", {
                     "name": f"{topic_name}",
                     "type": "topic"
                 }, self.kafka_rpc_service_name), 1, 3),
             AbsoluteTestItem(
                 f'Delete Topic {topic_name}',
-                lambda: self.rpk.delete_topic(topic=topic_name),
+                lambda: self.super_rpk.delete_topic(topic=topic_name),
                 partial(self.api_resource_match, "delete_topics", {
                     "name": f"{topic_name}",
                     "type": "topic"
                 }, self.kafka_rpc_service_name), 1),
             AbsoluteTestItem(
-                f'Create ACL',
-                lambda: self.rpk.allow_principal(principal="test",
-                                                 operations=["all"],
-                                                 resource="topic",
-                                                 resource_name="test"),
+                f'Create ACL', lambda: self.super_rpk.sasl_allow_principal(
+                    principal="test",
+                    operations=["all"],
+                    resource="topic",
+                    resource_name="test",
+                    username=self.security.user_creds[0],
+                    password=self.security.user_creds[1],
+                    mechanism=self.security.user_creds[2]),
```

Comment:
even though you're using `super_rpk`, this is going to use the regular user credentials.  You should be able to call `allow_principal` using `super_rpk` instead

Reply:
My point is that this is going to use the regular user credentials to perform the operation.  It doesn't set the acls for the username specified here.

Reply:
yup thats right +1